src/sketch.js

```diff
@@ -1,21 +1,28 @@
 /**
```

Comment:
The documentation section below belongs to the `setup` function, please move this section to above it.